### PR TITLE
Added workflows examples for hybrid and offline database

### PR DESCRIPTION
**Pull request by SeqLaz**
Feel free to edit or help me to correct any errors or omissions in this workflows examples

### CODE REVIEW

File: documentation/reference/qfieldcloud/system.en.md [hunk 1]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+#### Project Creation in QFieldCloud
+
+- Access [QFieldCloud](https:app.qfield.cloud/) and create a new project. Ensure that the project name follows the specified conventions: use underscores instead of spaces and avoid special characters. Provide all required information and specifications before finalizing the project creation by pressing the "Create" button.
+
+!![](../../assets/images/hybrid_workflow_001.png)
+
+#### Project Synchronization in QGIS
+
+- Open QGIS and navigate to the QFieldSync plugin.
+- Select "QFieldCloud Projects Overview" and choose the desired project, this case the new created project.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:114]:
```suggestion
- Select "QFieldCloud Projects Overview" and choose the desired project, in this case the newly created project.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 2]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+- Click "Synchronize Selected Cloud Project" to initiate project synchronization.
+
+!![](../../assets/images/hybrid_workflow_002.png)
+
+- Specify the directory for project synchronization. The default location (~/QField/cloud/project_name) is recommended.
+- Proceed by clicking "Next", then "OK".
+- Add layers from your PostgreSQL database, ensuring you use a connection established through a [pg_service.conf](../../how-to/pg-service.md) file.
+- Configure the project settings, including styles and forms as your requirements.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:122]:
```suggestion
- Configure the project settings, including styles and forms, as per your requirements.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 3]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+#### Project Synchronization
+
+- Return to QGIS and press "Synchronize Current Cloud Project" within QFieldSync.
+- Choose "Prefer Local" and "Perform Actions".
+
+!![](../../assets/images/hybrid_workflow_004.png)
+
+#### Survey and Data Collection
+
+- Once finished the Jobs, download and synchronize the project on devices designated for surveying.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:144]:
```suggestion
- Once you have finished the Jobs, download and synchronize the project on devices designated for surveying.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 4]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+- Start surveying, utilizing the project.
+- Changes made during the survey will be stored locally on the device.
+
+#### Data Upload to Cloud and Download the latest data
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:148]:
```suggestion
#### Data Upload to the Cloud and Download the latest data
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 5]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+
+!![](../../assets/images/hybrid_workflow_006.png)
+
 ### Offline database
+
 Offline editing mode with desktop synchronization
 
 !![Offline editing mode](../../assets/images/offline-schema.png)
+
+Offline Mode is designated to utilize GeoPackage to collect and consolidate the information then manually push the changes to the PostgreSQL:
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:166]:
```suggestion
Offline Mode is designated to utilize GeoPackage to collect and consolidate the information, then manually push the changes to PostgreSQL:
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 6]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+#### Project Creation in QFieldCloud in Offline mode
+
+- Access [QFieldCloud](https:app.qfield.cloud/) and create a new project. Ensure that the project name follows the specified conventions: use underscores instead of spaces and avoid special characters. Provide all required information and specifications before finalizing the project creation by pressing the "Create" button.
+
+!![](../../assets/images/offline_workflow_001.png)
+
+#### Project Synchronization in QGIS in Offline mode
+
+- Open QGIS and navigate to the QFieldSync plugin.
+- Select "QFieldCloud Projects Overview" and choose the desired project, this case the new created project.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:177]:
```suggestion
- Select "QFieldCloud Projects Overview" and choose the desired project, in this case the newly created project.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 7]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+- Click "Synchronize Selected Cloud Project" to initiate project synchronization.
+
+!![](../../assets/images/offline_workflow_002.png)
+
+- Specify the directory for project synchronization. The default location (~/QField/cloud/project_name) is recommended.
+- Proceed by clicking "Next", then "OK".
+- Add layers from your PostgreSQL database.
+- Configure the project settings, including styles and forms as your requirements.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:185]:
```suggestion
- Configure the project settings, including styles and forms, as per your requirements.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 8]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+
+#### Offline Data Conversion
+
+- Utilize the core QGIS plugin "OfflineEditing" (make sure to have it active).
+- Navigate to "Database > Offline Editing" and select "Convert to Offline Project...".
+
+!![](../../assets/images/offline_workflow_003.png)
+
+- Choose GeoPackage as the storage type for offline data.
+- Specify the offline data location within the cloud project directory. It's essential to save it in the location accessible to QFieldCloud project.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:195]:
```suggestion
- Specify the offline data location within the cloud project directory. It's essential to save it in a location accessible to QFieldCloud project.
```

File: documentation/reference/qfieldcloud/system.en.md [hunk 9]
```diff
@@ -95,11 +95,142 @@ project before being able to push new changes.
...
+
+- Download and synchronize the project on devices designated for surveying.
+- Conduct surveys in project.
+- Changes made during the survey will be stored locally on the device.
+
+#### Data Synchronization and Database Update
+
+- Upon returning to an area with internet connectivity push the changes made during the survey to the cloud by initiating synchronization.
+- Monitor the progress of synchronization jobs to ensure successful completion.
+- In QGIS, download the latest data and attachments of the project using QFieldSync.
```
Comment (lucienicolier) [inline documentation/reference/qfieldcloud/system.en.md:225]:
```suggestion
- In QGIS, download the latest data and attachments for the project using QFieldSync.
```

**Comment by lucienicolier** [comment]
@SeqLaz Nice job!

**Comment by suricactus** [comment]
Let's keep this open for a bit before merging. I am not sure why we still call this "Hybrid".

**Comment by suricactus** [comment]
We should not document things that suggest using QGIS offline editing, there was some kind of misunderstanding about this PR.

**Pull request closed without merging**